src/main/kotlin/com/cognifide/gradle/aem/instance/tail/InstanceAnalyzer.kt

```diff
@@ -31,32 +31,27 @@ class InstanceAnalyzer(
             logsChannel.consumeEach { log ->
                 options.logListener(log, instance)
 
-                if (options.incidentChecker(log)) {
+                if (options.incidentChecker(log, instance)) {
```

Comment:
probably there will be introduced `InstanceLog` (to avoid passing that extra arg)